index.html

```diff
@@ -1094,6 +1100,39 @@ <h2 id="create-payment-profile-dto">CreatePaymentProfileDto</h2>
                     </tr>
                 </tbody>
             </table>
+            <h2 id="create-payment-profile-dto-by-merchant-customer-id">CreatePaymentProfileByMerchantCustomerIdDto</h2>
+            <table>
+                <thead>
+                    <tr>
+                        <th>Name</th>
+                        <th>Description</th>
+                        <th>Type</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>PatientGuid</td>
+                        <td>Patients Guid</td>

```

Comment:
replace with this: "This is the Guid provided by the Nextech merchant. Sometimes referred to as 'Patient Guid'"